### PR TITLE
docs(netlify): write netlify setup and site mapping documentation

### PR DESCRIPTION
**Pull request by henczi-espirit**

### CODE REVIEW

File: docs/pages/Hosting/Netlify.adoc [hunk 1]
```diff
@@ -37,6 +37,9 @@ In order to follow the guide below without any problems, there are some precondi
 == Initial Setup Guide
 After this guide, you are the owner of your own PWA Template repository which builds are hosted in Netlify.
 If you are missing contextual information to any step, you might find it here https://docs.netlify.com/get-started/.
+To enhance the user experience of our Netlify documentation, we propose the addition of a "Deploy to Netlify" button. This button will enable users to swiftly deploy new websites from templates with a single click, streamlining the onboarding process for web developers and making it easier for them to deploy their applications on Netlify.
+
+image::https://www.netlify.com/img/deploy/button.svg[Deploy to Netlify, link="https://app.netlify.com/start/deploy?repository=https://github.com/netlify/netlify-statuskit"]
```
Comment (henczi-espirit) [inline docs/pages/Hosting/Netlify.adoc:42]: The repository should be the PWA template itself and not the the status kit of the netlify example

File: docs/pages/Hosting/Netlify.adoc [hunk 2]
```diff
@@ -37,6 +37,9 @@ In order to follow the guide below without any problems, there are some precondi
 == Initial Setup Guide
 After this guide, you are the owner of your own PWA Template repository which builds are hosted in Netlify.
 If you are missing contextual information to any step, you might find it here https://docs.netlify.com/get-started/.
+To enhance the user experience of our Netlify documentation, we propose the addition of a "Deploy to Netlify" button. This button will enable users to swiftly deploy new websites from templates with a single click, streamlining the onboarding process for web developers and making it easier for them to deploy their applications on Netlify.
```
Comment (henczi-espirit) [inline docs/pages/Hosting/Netlify.adoc:40]: please remove following sentence - it provides no benefit for the customer:
`"To enhance the user experience of our Netlify documentation, we propose the addition of a "Deploy to Netlify" button."`

Comment (henczi-espirit) [inline docs/pages/Hosting/Netlify.adoc:40]: I'd suggest to rewrite the second sentence 

"The following button will deploy a PWA template instance on Netlify, streamlining the onboarding process for web developers and making it easier for them to get started. You can use it and continue to the guide at chapter <<Configure and deploy Site>>

File: docs/pages/Hosting/Netlify.adoc
Comment (jodeleit-es) [inline docs/pages/Hosting/Netlify.adoc:1]: "Open the Site configuration view of" --> is there something missing here?
image

Comment (jodeleit-es) [inline docs/pages/Hosting/Netlify.adoc:1]: I would rather move the disclaimer to the beginning, before the current first chapter "Introduction"

File: docs/pages/Hosting/ProjectToSiteMapping.adoc
Comment (jodeleit-es) [inline docs/pages/Hosting/ProjectToSiteMapping.adoc:1]: Small change suggestion (twice in the 
"[...] context you’ve got your [..] stage covered." --> "[..] context, you have your staging stage covered."

File: docs/pages/Hosting/ProjectToSiteMapping.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+[WARNING]
+====
+Keep in mind that a pull request from staging to production will trigger a deploy preview using a development environment. 
+Unfortunately there is no other way to set this up as netlify only allows one set of variables for the deploy preview deployment context.
+====
+
+== Continuous Deployment
+
+We recommend setting the following properties here
+- Production branch: This property states which branch should be used to deploy the site.
```
Comment (jodeleit-es) [inline docs/pages/Hosting/ProjectToSiteMapping.adoc:38]: This is not rendered as bulleted list...

File: docs/pages/Hosting/ProjectToSiteMapping.adoc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+We recommend setting the following properties here
+- Production branch: This property states which branch should be used to deploy the site. 
+This should be set to your main branch on github. 
+Or whichever branch contains your production level code.
+- Branch deploys: Allows to define whether any branches on github should also deployed as part of this site. 
+You can add individual branches to be deployed (eg. staging). 
+We would recommend adding your staging or qa branch to the list of individual branches.
+- Deploy previews: A preview is deployed when a pull request is made against the production branch 
+and redeployed whenever a new commit gets pushed onto the branch. 
+Deploy previews are not only useful because they let you test your current development branch in the netlify environment.
```
Comment (jodeleit-es) [inline docs/pages/Hosting/ProjectToSiteMapping.adoc:46]: Either you drop the "not only" here or you add something like "also" to the following sentence ;)

File: docs/pages/Hosting/ProjectToSiteMapping.adoc [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+image:edit_environment_variables2.png[Environment variable configuration options]
+Keep in mind that if you want to map your FirstSpirit project stages to the different deployment 
+contexts of netlify you need to choose the option "Different value for each deploy context" for most of your environment variables.
+
+=== NITRO_PRESET
+
+You can use this variable to deploy the site using netlify edge functions.
+In order to do this, set this variable to `netlify-edge`.
+Edge functions are considerably faster than normal functions but have a tighter restriction on app performance.
+You can read all about these limits in the https://docs.netlify.com/edge-functions/limits/[netlify docs]
```
Comment (jodeleit-es) [inline docs/pages/Hosting/ProjectToSiteMapping.adoc:68]: Period "." at the end of the sentence is missing

File: docs/pages/Hosting/Netlify.adoc
Comment (Dramaturco) [reply docs/pages/Hosting/Netlify.adoc:1]: That would be inconsistent with the other disclaimers. But I could move those up as well. I agree, a disclaimer should be read first so it stays in the head while reading everything else.

Comment (Dramaturco) [reply docs/pages/Hosting/Netlify.adoc:1]: Yes the word netlify is missing there

Comment (jodeleit-es) [reply docs/pages/Hosting/Netlify.adoc:1]: :+1:

**Review by jodeleit-es — APPROVED** [review]
:+1:

**Pull request merged**